src/Consolonia.Themes/Templates/Controls/CalendarItem.axaml

```diff
@@ -33,7 +32,7 @@
                                 <Style Selector="Button.CalendarHeader:pressed  /template/ ContentPresenter">
                                     <!--todo: needs some color may be-->
                                     <Setter Property="Background"
-                                            Value="{drawing:ConsoleBrush Mode=Transparent}" />
+                                            Value="{DynamicResource ThemeTransparentBrush}" />
```

Comment:
do you think there is ever a need for a visible transparent brush that would be themed?  Why not just Brushes.Transparent?

Reply:
My only thought here is it can be required to be defined by another type, not Brush. But probably does not matter now. I'm good to make a separate PR for this just not to mix